### ISSUE TITLE
add qk norm with pretrained model， 

### ISSUE THREAD

**Issue by lbwang2006** [hpcaitech/Open-Sora#543]
add qk norm with pretrained text to image model， the generated images failed with invalid content, the training step needs to 10000 step?

**Comment by JThh** [comment]
Could @zhengzangw please take a look?

**Comment by JThh** [comment]
Can @lbwang2006 also provide more information on how and where you added QK normalization?

**Comment by zhengzangw** [comment]
Not quite sure about your question. We do use 10k steps to adapt for qk-norm.

**Comment by lbwang2006** [comment]
> Not quite sure about your question. We do use 10k steps to adapt for qk-norm.

retrain the whole parameters after qk-norm inserted or part of it？Is there any configure file in the codebase？
